controller/transfer/manager.py

```diff
@@ -57,9 +57,11 @@ def get_upload_credentials_and_id(
     file_type: str,
     file_import_options: str,
     upload_type: str,
-):
+    key: str,
```

Comment:
wouldn't this be optional or am i missing something?

Reply:
Yeah should be optional